examples/bls-multisig/rust/src/main.rs

```diff
@@ -0,0 +1,100 @@
+use alloy::{primitives::U256, providers::ProviderBuilder, sol, sol_types::SolValue};
+use blst::min_pk::{AggregateSignature, SecretKey, Signature};
+use rand::RngCore;
+use BLS::G2Point;
+
+sol! {
+    #[derive(Debug, Default, PartialEq, Eq, PartialOrd, Ord)]
+    #[sol(rpc)]
+    BLSMultisig,
+    "../../../out/BLSMultisig.sol/BLSMultisig.json"
+}
+
+impl From<[u8; 96]> for BLS::G1Point {
```

Comment:
makes me think we should create an alloy-bls package which also includes a tx signer for the AlphaNet payloads for the BatchContract -- cc @jxom